adal/src/main/java/com/microsoft/aad/adal/Discovery.java

```diff
@@ -59,7 +59,7 @@
  * sends common as a tenant name. Discovery checks only authorization endpoint.
  * It does not do tenant verification. Initialize and call from UI thread.
  */
-final class Discovery {
+class Discovery {
```

Comment:
Is there a reason this is no longer `final`?

Reply:
in Ad-accounts, there's a class name "BrokerDiscovery" that extends the Discovery class, so it could not be final